tests/simulation/test_simulation.py

```diff
@@ -52,10 +54,13 @@ async def simulation(self):
     @pytest.mark.asyncio
     async def test_simulation_1(self, simulation, extra_node):
         node1, node2, _, _, _, _, _, _, _, sanitizer_server, server1 = simulation
-        await server1.start_client(PeerInfo(self_hostname, uint16(21238)))
+
+        node1_port = node1.full_node.config["port"]
+        node2_port = node2.full_node.config["port"]
+        await server1.start_client(PeerInfo(self_hostname, uint16(node2_port)))
         # Use node2 to test node communication, since only node1 extends the chain.
-        await time_out_assert(1500, node_height_at_least, True, node2, 7)
-        await sanitizer_server.start_client(PeerInfo(self_hostname, uint16(21238)))
+        await time_out_assert(600, node_height_at_least, True, node2, 7)
+        await sanitizer_server.start_client(PeerInfo(self_hostname, uint16(node2_port)))
```

Comment:
We previously had issues with low timeouts here, some CIs can take a really long time. note that this is running the real timelord and farmer all in the same process.

Reply:
granted, this is unrelated. While testing though, it was frustrating to have this test take 1500 seconds (as well as `test_daemon.py`) before failing. In all my successful local runs, it took 30 seconds to complete. I figured 20x margin would be enough.